tiny fixes to my stuff. (Sorry! >~<)

nullbear opened the pull request:
makes xeno computer deconstructible again.

From grenade to chemical grenade. This has no effect but to make it so
that tablecrafting recipes calling for chemgrenades specify that its a
grenade casing it needs, not any grenade/flashbang/etc.

Tweaks syndicatebomb recipe so that any matter bin will do, making it
possible to build without rnd's participation whatsoever.

Using a tier one matter bin makes it effectively LESS useful than a
grenade, however.

:cl: nullbear
fix: xenobio console is deconstructable again.
tweak: syndibomb recipe calls for any matter bin, scaling effectiveness with tier.
/:cl:

optimumtact commented:
code/game/machinery/syndicatebomb.dm:414:error: obj: undefined var

code/game/machinery/syndicatebomb.dm:414:error: item: undefined var

code/game/machinery/syndicatebomb.dm:414:error: weapon: undefined var

code/game/machinery/syndicatebomb.dm:414:error: stock_parts: undefined var

code/game/machinery/syndicatebomb.dm:414:error: matter_bin: undefined var

nullbear commented:
whoops.

GunHog commented on code/modules/crafting/recipes.dm:
```diff
@@ -344,10 +344,10 @@
 		/obj/item/weapon/grenade/chem_grenade = 2
 	)
-	parts = list(/obj/item/weapon/grenade/chem_grenade = 2)
+	parts = list(/obj/item/weapon/stock_parts/matter_bin = 1, /obj/item/weapon/grenade/chem_grenade = 2)
```
Does the same matter bin used in the bomb's construction transfer to the bomb itself? Is there a way to swap out the matter bin after the bomb is constructed?

nullbear replied:
the same matter bin transfers to it, but payloads aren't currently deconstructable so i just get rid of that information after using it.

nullbear commented:
@phil235 Thanks for pointing that out. Sorry m8

Jordie0608 commented on code/game/machinery/syndicatebomb.dm:
```diff
@@ -411,6 +411,9 @@
 /obj/item/weapon/bombcore/chemical/CheckParts()
 	// Using different grenade casings, causes the payload to have different properties.
+	var/obj/item/weapon/stock_parts/matter_bin/MB = locate(/obj/item/weapon/stock_parts/matter_bin) in src
+	max_beakers += MB.rating	// max beakers = 2-5.
```
You should add a `if(MB)` check just in case

nullbear replied:
k

Jordie0608 commented on code/game/objects/items/weapons/grenades/chem_grenade.dm:
```diff
@@ -240,20 +240,21 @@
-		if(unit_spread < 100)
+		if(unit_spread<25)
 			unit_spread += 5
+		if(unit_spread<100)
```
Won't this also be triggered if `unit_spread` is less than 25?

nullbear replied:
Its a switch statement, not an if block, but i'll test real quick and make sure.

Jordie0608 replied:
There's no `switch(unit_spread)` here

nullbear committed:
tiny fixes to little things

allows deconstruction of xenobio computer again.

renames chem grenades to chem grenades.

makes advanced grenades scale up their unit thing.

makes chemical payload scale up with matter bin used in construction,
and more accessible in general
